cmd/root.go

```diff
@@ -70,7 +71,6 @@ func init() {
 	cobra.OnInitialize(initConfig)
 	RootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "deco config file -- _not_ the control file (default is $HOME/.deco.yaml)")
 	RootCmd.PersistentFlags().StringVarP(&baseDir, "dir", "d", "", "Base directory for filtered files/templates")
-	RootCmd.PersistentFlags().StringArrayVarP(&httpHeaders, "header", "H", []string{}, "Pass a custom header to server")
```

Comment:
Don't we need this anymore, or it's just not a persistent flag?

Reply:
we don't need it on the root command, it still exists on the run, show, validate commands.